From 1e4d0d567a72d444dff1f90bb9c1712212d6e07e Mon Sep 17 00:00:00 2001
From: Fushan Wen <qydwhotmail@gmail.com>
Date: Sat, 18 Mar 2023 16:31:57 +0800
Subject: [PATCH] applet: add missing function for "Show virtual devices" menu
 item

BUG: 465996
FIXED-IN: 5.27.4
(cherry picked from commit 5430caf2483203de5f3047408582bec953aa8946)
---
 applet/contents/ui/main.qml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/applet/contents/ui/main.qml b/applet/contents/ui/main.qml
index aa595204..f65ec85d 100644
--- a/applet/contents/ui/main.qml
+++ b/applet/contents/ui/main.qml
@@ -754,6 +754,10 @@ Item {
         KQCAddons.KCMShell.openSystemSettings("kcm_pulseaudio");
     }
 
+    function action_showVirtualDevices() {
+        Plasmoid.configuration.showVirtualDevices = !Plasmoid.configuration.showVirtualDevices;
+    }
+
     Component.onCompleted: {
         MicrophoneIndicator.init();
 
-- 
GitLab

